bin/nativetests/JsRTApiTest.cpp

```diff
@@ -0,0 +1,1603 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "stdafx.h"
+#include "catch.hpp"
+#include <process.h>
+
+#pragma warning(disable:4100) // unreferenced formal parameter
+
+namespace JsRTApiTest
```

Comment:
These tests are transformed from our full repo. So the intent of the tests have not been changed.